release_notes/ocp-4-19-release-notes.adoc

```diff
@@ -1933,12 +1922,18 @@ With this release, availability sets are no longer modified after creation, allo
 
 * Previously, there was no event logged when an error occurred from a failed conversion from ingress to route. With this update, an error for a failed conversion is logged. (link:https://issues.redhat.com/browse/OCPBUGS-29354[OCPBUGS-29354])
 
-* Previously, when you created a virtual machine (VM) and no IP addresses existed in the IP pool, the VM did not start. An error message was generated in the `virt-launcher-<vm-name>` pod, but the message was unclear as to the source of the issue. With this release, for this situation where no IP addresses exist in the IP pool, the `virt-launcher-<vm-name>` pod includes a clear error message that is similar to the following example:
+* Previously, the PowerVS installer used a hard-coded list of supported machine types. However, this list was not always updated as new types were added. With this release, datacenter is queried to get the current list of supported types. (link:https://issues.redhat.com/browse/OCPBUGS-49940[OCPBUGS-49940])
+
+* Previously, when a RootDiskHint was defined and the installation failed with the error "Requested installation disk is not part of the host's valid disks", it was difficult to determine the valid disk names that could be used as the hint. With this release, logging was added for the list of acceptable disks so that a user can quickly determine what the root disk hint should be. (link:https://issues.redhat.com/browse/OCPBUGS-43578[OCPBUGS-43578])
```

Comment:
```suggestion
* Previously, when a `RootDiskHint` resource was defined and the installation failed with the error "Requested installation disk is not part of the host's valid disks", it was difficult to determine the valid disk names that could be used as the hint. With this release, logging was added for the list of acceptable disks so that a user can quickly determine what the root disk hint should be. (link:https://issues.redhat.com/browse/OCPBUGS-43578[OCPBUGS-43578])
```